airbyte-cdk/python/airbyte_cdk/sources/declarative/__init__.py

```diff
@@ -1,3 +1,39 @@
 #
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
+
+from .yaml_declarative_source import YamlDeclarativeSource
+from .auth.declarative_authenticator import DeclarativeAuthenticator
+from .checks.connection_checker import ConnectionChecker
+from .decoders.decoder import Decoder
+from .extractors import HttpSelector, RecordFilter, RecordSelector
+from .incremental import DeclarativeCursor
+from .interpolation import InterpolatedString, InterpolatedBoolean, InterpolatedMapping
+from .migrations.state_migration import StateMigration
+from .requesters import Requester
+from .requesters.paginators import Paginator
+from .retrievers import Retriever
+from .schema import SchemaLoader
+from .stream_slicers import StreamSlicer
+from .transformations import RecordTransformation
+from .declarative_stream import DeclarativeStream
+
+__all__ = ["ConnectionChecker",
```

Comment:
expose the interfaces of the declarative module since connector developer can create custom class inheriting them. Changing these interfaces should be considered a breaking change